### ISSUE TITLE
Allow `TFuncKey` to be used without specifying the namespace, in the same way `TFunction` and `useTranslation` work

### ISSUE THREAD

**Issue by raynor85** [i18next/react-i18next#1261]
## 🚀 Feature Proposal

Allow the type `TFuncKey` to resolve the default namespace if no generics are passed in. This is already the behaviour of `TFunction` and other exported types.

## Motivation

It is very common to import `TFuncKey` type to ensure that string literals are valid translation keys. Repeating the namespace every time is not great, we could just default it to the default namespace

## Example

At the moment is possible to do this with `TFunction` and with `useTranslation`:

```
const {t} = useTranslation(); // it is like writing const {t} = useTranslation<"myDefaultNamespace">();

const getTranslation = (t: TFunction) => {...} // it is like writing const getTranslation = (t: TFunction<"myDefaultNamespace">) => {...}
```

but I can't do the same for `TFuncKey`:

```
const key: TFuncKey = 'my.translation.value', // error: Generic type 'TFuncKey' requires between 1 and 2 type arguments.
```

**Comment by raynor85** [comment]
Created a PR for this. It looks like the build fails, but it has nothing to do with the PR itself https://github.com/i18next/react-i18next/pull/1262

**Comment by raynor85** [comment]
This has been resolved as the PR was merged and published in react-i18next@11.8.8
